depends/packages/native_qt.mk

```diff
@@ -148,5 +150,5 @@ endef
 
 define $(package)_postprocess_cmds
   rm -rf doc/ && \
-  mv -t .. translations/
+  mv translations/ ..
```

Comment:
Had to look this up, but apparently the only thing `-t` does is to set the target directory explicitly. So `mv translations ..` is equivalent.